src/Symfony/Component/WebLink/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symfony/weblink",
```

Comment:
We indeed use a caret in other components (`event-dispatcher`, `http-foundation`), but as @fabpot wants.